src/utils/utils.ts

```diff
@@ -69,3 +69,83 @@ export const createDatesArray = (
   }
   return datesArray
 }
+
+interface CleanedEarn {
+  _id: number
+  data: [
+    {
+      minFee: number
+      maxFee: number
+      dayCount: number
+      memCount: number
+      minDelay: number
+      maxDelay: number
+      minMinutes: number
+      maxMinutes: number
+    }
+  ]
+}
+
+interface EarnAllInfo {
```

Comment:
I believe you can extend an `Interface`. Check out this link and search for `Extending Types`.